src/api/server/common.rst

```diff
@@ -453,7 +453,9 @@
 .. http:get:: /_db_updates
     :synopsis: Return the server changes of databases
 
-    Returns a list of all database events in the CouchDB instance.
+    Returns a list of all database events in the CouchDB instance. The
+    existence of the ``_global_changes`` database is required to use this
+    endpoint.
```

Comment:
This is not true.

`/_db_updates` will function if `global_changes` is not present, but it will only show current data, not historical. That is, you cannot provide a `since` token and see historical values; it will act as if you always specify `?since=now`.

Reply:
Interesting, because the endpoint does not show any data for me till I do not create the `_global_changes` database. 

```
curl --location --request GET 'localhost:5984/_db_updates' \
--header 'Authorization: ***'
```
Gives me a 404 with:
```json
{
    "error": "not_found",
    "reason": "Database does not exist."
}
```
@wohali what do you think, should I report this first?

Reply:
You have to specify `continuous=true` as well, I believe, then try to create a DB while you're listening. Try that.

Reply:
hi,

It is true that you need the backing db ("_global_changes") for the _db_updates endpoint to work at all, regardless of feed type.

Reply:
```cluster_system_dbs() ->
    ["_users", "_replicator"].``` from https://github.com/apache/couchdb/blob/master/src/setup/src/setup.erl#L77-L78

Reply:
Thanks, I was mistaken, remembering how this worked before the backing store. 

Resolved.